nativelib/src/main/resources/scala-native/time_millis.c

```diff
@@ -1,6 +1,8 @@
+#include <time.h>
```

Comment:
I recommend a short comment describing what this method
(and nanosecond companion) are supposed to do?

Are they intended to have the same semantics as the Javalib
near namesakes or something different.  

Knowing the intended behavior, even a line or two makes 
it easier to review the code for correctness.  When the code
is the specification, obviously it is correct.

I think the practice of erecting a guidon is pertinent across the case, but is especially useful in this case, where one
method uses "time since the Epoch" and the other uses "time since some arbitrary starting time".
 
The intended meaning of the methods may be obvious now
(to everybody but laggard me) but will they be to another generation of maintainers in six months?

Reply:
Now I am wondering if these should be in javalib or they are used elsewhere.